shared/menubar/index.desktop.js

```diff
@@ -153,7 +188,18 @@ class MenubarRender extends Component<Props, State> {
       onRekey: this.props.onRekey,
     }
 
-    const badgeTypes: Array<Tab> = [peopleTab, chatTab, fsTab, teamsTab]
+    const badgeTypesInHeader: Array<Tab> = [peopleTab, chatTab, fsTab, teamsTab]
+    const badgesInMenu: Set<string> = new Set([
+      ...(flags.walletsEnabled ? [walletsTab] : []),
+      gitTab,
+      devicesTab,
+      settingsTab,
+    ])
+    const badgeCountInMenu = Object.entries(this.props.badgeInfo).reduce(
```

Comment:
I'm curious why a `reduce` is used here rather than counting on `badgesInMenu`. Do we have situations where things exist in `badgesInMenu` but not `this.props.badgeInfo`?

Reply:
Good point. I'll change it.